packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -198,6 +198,14 @@ module.exports = (themeOptions = {}) => {
           exclude: [],
         },
       },
+      pluginOptions.hubspotTrackingCode && {
+        resolve: 'gatsby-plugin-hubspot',
+        options: {
+          trackingCode: pluginOptions.hubspotTrackingCode,
+          respectDNT: true,
```

Comment:
Please set this to false like we do in GA - there's no point in having differing config and in analytics it's important that all sites use the same config (all other ct sites are not setting the respectDNT flag, too).